### PR TITLE
gist: add two examples

### PR DESCRIPTION
**Pull request by jasonxia23**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      "command: add page" for new pages, or "command: description of changes" for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by waldyrious** [comment]
@jasonxia23 it looks like one of our points in the PR checklist ("The PR is appropriately titled") is confusing. The intended meaning is that this particular PR, for instance, would be titled `gist: update description`. Is there a wording for that instruction that you would consider clearer? Please let us know. /cc @agnivade.

File: pages/common/gist.md [hunk 1]
```diff
@@ -18,6 +22,10 @@
 
 `gist -l`
 
+- List all gists for user:
```
Comment (sbrl) [inline pages/common/gist.md]: Perhaps "the currently logged in user"?

Comment (jasonxia23) [reply pages/common/gist.md]: Certainly, it's more clear.

File: pages/common/gist.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 > Upload code to https://gist.github.com.
 
+- Authenticate gist on this computer:
```
Comment (waldyrious) [inline pages/common/gist.md]: I'm not sure I understand this description. @agnivade, @sbrl, how did you interpret this sentence?

Comment (jasonxia23) [reply pages/common/gist.md]: It is from `gist --help`. Expect A more accurate description. What about
> Login in gist on this computer.

Comment (sbrl) [reply pages/common/gist.md]: @jasonxia23 Yeah, that's a good idea!

Comment (waldyrious) [reply pages/common/gist.md]: I like that too :+1:

**Comment by sbrl** [comment]
Thanks 😺

**Pull request merged**